test/f.jl

```diff
@@ -0,0 +1,37 @@
+using HypothesisTests, Test
+using HypothesisTests: default_tail
+
+@testset "F-tests" begin
+    @testset "Basic variance F-test" begin
+        Random.seed!(12)
+        y1_h0 = 4 .+ randn(500)
+        y2_h0 = 4 .+ randn(400)
+
```

Comment:
Can you repeat the test also with vectors switched?

Reply:
Done

Reply:
But it is not used in computing a p value... it is not providing the functionality it suggests